lib/class-wp-theme-json.php

```diff
@@ -523,10 +523,7 @@ private static function process_key( $key, &$input, $schema ) {
 	 * @return array|null The settings subtree.
 	 */
 	private static function extract_settings( $context ) {
-		if (
-			! array_key_exists( 'settings', $context ) ||
-			empty( $context['settings'] )
-		) {
+		if ( ! isset( $context['settings'] ) ) {
```

Comment:
This is a slightly different behavior as it won't return when the value is falsy. Should it be updated to:
```suggestion
		if ( empty( $context['settings'] ) ) {
```
instead?

Reply:
There are two more places, that had a similar check.